File: bindings/java/src/org/raml/amf/generators/OpenAPIGenerator.java
package org.raml.amf.generators;

/**
 * Created by antoniogarrote on 04/05/2017.
 */


/**
 * Serialises the AMF model as an OpenAPI JSON document
 */
public class OpenAPIGenerator extends BaseGenerator {

    @Override
    protected String generatorConstructor() {
        return "->OpenAPIGenerator";
    }
}

File: bindings/js/src/generators/OpenAPIGenerator.ts
/**
 * Created by antoniogarrote on 05/05/2017.
 */

import {BaseGenerator} from "./BaseGenerator";
import {Clojure} from "../Clojure";

/**
 * Serialises the AMF model as an OpenAPI JSON document
 */
export class OpenAPIGenerator extends BaseGenerator {
    protected generator(): any {
        return new Clojure.amf.__GT_OpenAPIGenerator();
    }
}